src/main/java/com/rainforestcommerce/rcdb/models/ProductQuantityPrice.java

```diff
@@ -0,0 +1,51 @@
+package com.rainforestcommerce.rcdb.models;
+
+/**
+ * Represents a product with a quantity and unit price.
+ * Used to represent a product in a store's inventory and
+ * a product in a purchase.
+ * @author Graham
+ *
+ */
+public class ProductQuantityPrice extends Product {
+
+    private long purchaseId;
+    private float unit_price;
+    private int quantity;
+
+
+    public ProductQuantityPrice(long purchaseId, long productId, int unit_price, int quantity, Product product){
```

Comment:
I'd rename this to something more semantic relating to the store's inventory. e.g. `StoreInventoryItem`.